.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
       - *attach_workspace
       - run:
           name: "deploy"
-          command: "sudo npm i -g serverless && sls deploy -s mosaic-prod --CLIENT_EMAIL $CLIENT_EMAIL --PRIVATE_KEY $PRIVATE_KEY --TEMPLATE_DOCUMENT_ID $TEMPLATE_DOCUMENT_ID --TITLE $TITLE"
+          command: "sls deploy -s mosaic-prod --CLIENT_EMAIL $CLIENT_EMAIL --PRIVATE_KEY $PRIVATE_KEY --TEMPLATE_DOCUMENT_ID $TEMPLATE_DOCUMENT_ID --TITLE $TITLE"
```

Comment:
You may need to move this to a script in `package.json`, something like:

```json
"scripts": {
  "deploy": "sls deploy"
}
```

Then in your Circle config have:

```yml
command: "yarn deploy -s mosaic-prod --CLIENT_EMAIL $CLIENT_EMAIL --PRIVATE_KEY $PRIVATE_KEY --TEMPLATE_DOCUMENT_ID $TEMPLATE_DOCUMENT_ID --TITLE $TITLE"
```

This will make `yarn` responsible for sourcing the `sls` binary from `node_modules`.

Reply:
I see. Thanks 🙌🏾